src/packet_pipeline.cpp

```diff
@@ -104,16 +104,26 @@ DepthPacketProcessor *PacketPipeline::getDepthPacketProcessor() const
 }
 
 CpuPacketPipeline::CpuPacketPipeline()
-{ 
-  comp_->initialize(new TurboJpegRgbPacketProcessor(), new CpuDepthPacketProcessor());
+{
+#ifdef LIBFREENECT2_WITH_VT_SUPPORT
+  RgbPacketProcessor *rgb = new VTRgbPacketProcessor();
+#else
+  RgbPacketProcessor *rgb = new TurboJpegRgbPacketProcessor();
+#endif
```

Comment:
I don't think returning NULL is well intentioned. But the configuration in CMake makes it never return NULL, so I'm ok with this.